docs/book/src/developer/providers/v1alpha2-to-v1alpha3.md

```diff
@@ -371,6 +371,83 @@ commonLabels:
   cluster.x-k8s.io/v1beta1: v1alphaX,v1beta1
 ```
 
+# Upgrade to CRD v1
```

Comment:
Should we put this at the very top? Seems might be the first thing to tackle probably

Reply:
the item right before it `update config folder` is a prerequisite, so I put it right after that one